homeassistant/components/venstar/sensor.py

```diff
@@ -0,0 +1,135 @@
+"""Representation of Venstar sensors."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Any
+
+from homeassistant.components.sensor import (
+    DEVICE_CLASS_HUMIDITY,
+    DEVICE_CLASS_TEMPERATURE,
+    STATE_CLASS_MEASUREMENT,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE, TEMP_CELSIUS, TEMP_FAHRENHEIT
+
+from . import VenstarDataUpdateCoordinator, VenstarEntity
+from .const import DOMAIN
+
+
+@dataclass
+class VenstarSensorEntityDescription(SensorEntityDescription):
+    """Base description of a Sensor entity."""
+
+
+HUMIDITY_SENSOR_DESCRIPTION = VenstarSensorEntityDescription(
+    key="humidity",
+    device_class=DEVICE_CLASS_HUMIDITY,
+    state_class=STATE_CLASS_MEASUREMENT,
+    native_unit_of_measurement=PERCENTAGE,
+)
+TEMPERATURE_SENSOR_DESCRIPTION = VenstarSensorEntityDescription(
+    key="temperature",
+    device_class=DEVICE_CLASS_TEMPERATURE,
+    state_class=STATE_CLASS_MEASUREMENT,
+)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities) -> None:
+    """Set up Vensar device binary_sensors based on a config entry."""
+    coordinator = hass.data[DOMAIN][config_entry.entry_id]
+    entities: list[Any] = []
```

Comment:
Please avoid using Any since we know what kind of entities can be stored here.

Reply:
Without that, mypy complains:

error: Argument 1 to "append" of "list" has incompatible type "VenstarTemperatureSensor"; expected "VenstarHumiditySensor" [arg-type]

or will I have to list each of those as a possibility?  Won't that be obnoxious when I add more sensor types?

Reply:
This might work 

```python3
diff --git a/homeassistant/components/venstar/sensor.py b/homeassistant/components/venstar/sensor.py
index 45121d34b7..12a504ee0b 100644
--- a/homeassistant/components/venstar/sensor.py
+++ b/homeassistant/components/venstar/sensor.py
@@ -13,6 +13,7 @@ from homeassistant.components.sensor import (
 )
 from homeassistant.config_entries import ConfigEntry
 from homeassistant.const import PERCENTAGE, TEMP_CELSIUS, TEMP_FAHRENHEIT
+from homeassistant.helpers.entity import Entity
 
 from . import VenstarDataUpdateCoordinator, VenstarEntity
 from .const import DOMAIN
@@ -39,7 +40,7 @@ TEMPERATURE_SENSOR_DESCRIPTION = VenstarSensorEntityDescription(
 async def async_setup_entry(hass, config_entry, async_add_entities) -> None:
     """Set up Vensar device binary_sensors based on a config entry."""
     coordinator = hass.data[DOMAIN][config_entry.entry_id]
-    entities: list[Any] = []
+    entities: list[Entity] = []
 
     sensors = coordinator.client.get_sensor_list()
     if not sensors:
```